app/views/declaration/destinationCountries/change_routing_country.scala.html

```diff
@@ -73,7 +73,7 @@
         @govukFieldset(Fieldset(
             legend = Some(Legend(
                 content = Text(messages(s"declaration.${page.id}.question")),
-                classes = "govuk-fieldset__legend--xl",
+                classes = "govuk-fieldset__legend--l",
```

Comment:
Why did you not use the constant?  Then, when all pages are GDS, we only have one place to change to get page titles to the right size.

Reply:
Sorry!!! :) Hi did not read your comments

Reply:
I will do that... tomorrow :) Too tired today

Reply:
👍 - make sure you squash/rebase your commits!  :-)